examples/simulation/simulate_toy_detector.cpp

```diff
@@ -90,6 +89,12 @@ int simulate(std::string output_directory, unsigned int events,
 
     sim.run();
 
+    // Create detector file
+    auto writer_cfg = detray::io::detector_writer_config{}
```

Comment:
Just out of curiosity: You already have the toy detector file available for traccc, right? Why are you writing it out again?